app/lib/backend/model_extensions/cinstance.rb

```diff
@@ -39,8 +39,7 @@ def update_backend_application
                                               :state      => state,
                                               :plan_id    => plan.id,
                                               :plan_name  => plan.name,
-                                              :redirect_url => redirect_url,
-                                              :user_required => end_user_required.nil? ? plan.end_user_required : end_user_required  )
```

Comment:
Is it OK at both pisoni and apisonator if we don't pass this attribute anymore?

Reply:
https://github.com/3scale/porta/pull/1648#discussion_r385117376 @guicassolato